policyengine_us/parameters/gov/states/dc/tax/income/credits/ptc/fraction_nonelderly.yaml

```diff
@@ -23,7 +27,11 @@ brackets:
     amount:
       2021-01-01: 0.05
   - threshold:
-      2021-01-01: 56_200
-      2022-01-01: 57_600
+      values:
+        2021-01-01: 56_200
+        2022-01-01: 57_600
+        2023-01-01: 61_300
+      metadata:
```

Comment:
need to add rounding:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/3a452346-eb1b-4a03-9856-5c4fe4a241c2)